source/src/main/java/org/cerberus/engine/execution/impl/SeleniumServerService.java

```diff
@@ -584,7 +587,19 @@ private DesiredCapabilities setCapabilityBrowser(DesiredCapabilities capabilitie
                  */
                 ChromeOptions options = new ChromeOptions();
                 // Maximize windows for chrome browser
+               
```

Comment:
Is it possible to avoid duplication code, and just precise `--window-size` and `--headless` only for chrome ?

It will allow to delete condition for chrome on previous changement.